ds-cv-inference/mqtt/mqtt.go

```diff
@@ -67,7 +68,12 @@ var commandTopicFunction MQTT.MessageHandler = func(client MQTT.Client, msg MQTT
 
 	fmt.Printf("received message: %v+", edgeXMessage)
 
-	switch edgeXMessage["cmd"] {
+	words := strings.Split(msg.Topic(), "/")
+	cmd := words[2]
+	uuid := words[4]
```

Comment:
it could have index out of bound if msg is not well-formed or expected, need to add input validation?

Reply:
done